android/src/main/java/com/reactlibrary/RNWalkme.java

```diff
@@ -1,5 +1,5 @@
 
-package com.homeis;
+package com.reactlibrary;
```

Comment:
@adylevy this must be changed because you state the `com.reactlibrary` on the AndroidManifest, at the moment automatic integrations isn't working because of the wrong import statement.

Reply:
Hi!
Can you explain more? Our app has loads of 3rd party react-native plugin and non of them sits under com.reactlibrary 

Some examples:
MixPanel - com.kevinejohn.RNMixpanel.RNMixpanel
Fabric - com.smixx.fabric.FabricPackage
Facebook SDK - com.facebook.FacebookSdk

So I am confused about the need to sit under a specific package name.

Reply:
@adylevy see this file:
https://github.com/Homeisapp/react-native-walkme/blob/development/android/src/main/AndroidManifest.xml

You've stated the package for the plugin, but the file sits on different folder.

Try to integrate the plugin from a fresh new project and you'll see that automatic linkage is not working.

Reply:
Ok, so changing the AndroidManifest to properly have com.homeis will solve it right?

Reply:
yes